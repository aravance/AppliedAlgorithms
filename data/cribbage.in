5H 5D 7D JD 8D
5H 5D 6C 7S 8H
5C 4C 6C JS TS
3S 3D 3C 9D 6D
AC 2D 3H 4S 5S
AS 2S 3S 4S 5H
AS 2S 3S 4S 5S
AS 2S 3S 4H 5S
9H TH JH QH KH
6D 7S 7C 8C 8D
2D 3C 4S 4C 4H
2S 3C 3S 3D 4H
6D 3C 3S 3D 6C
5C 4D 4S 4C 6C
5C 5D 5H JD JS
5C 5H JD JS 5D
5C 5D 5S 4C 6C
JS 5C 5D 5H 5S
5C 5D 5H 5S JS
TH JH QH KH AH
2C 3C 5C 6D 8C
2H 4H 6C TH JH
6C 7D TH JS KC
0
