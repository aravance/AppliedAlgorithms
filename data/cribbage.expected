9
12
10
16
7
11
12
7
11
24
17
17
18
21
22
23
23
29
28
10
2
0
0
